src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -90,7 +92,8 @@ class ServerListWindow final : public Window
 
     void OnOpen() override
     {
-        window_server_list_widgets[WIDX_PLAYER_NAME_INPUT].string = _playerName;
+        _playerName = gConfigNetwork.PlayerName;
+        window_server_list_widgets[WIDX_PLAYER_NAME_INPUT].string = const_cast<utf8*>(_playerName.c_str());
```

Comment:
This doesn't seem very safe. The buffer that `.c_str()` returns is not guaranteed to stay in place when the string changes. When it moves, this widget will point to garbage.